ops/ansible/roles/bfd-server/README.md

```diff
@@ -17,13 +17,9 @@ This role is highly configurable, though it tries to provide reasonable defaults
 
 Variables that must be defined to use the role:
 
-* `data_server_artifacts_mode`: Set to "`s3`" if the WAR and app server should be copied from an AWS S3 bucket or "`local`" if they should be copied from a path local to the Ansible management host.
-* `data_server_artifacts_s3_bucket`: The name of the S3 bucket containing the artifacts to be deployed. Must be set if `data_server_artifacts_mode` is "`s3`".
-* `data_server_appserver_installer_name`: The name of the app server installation bundle to be deployed.
-* `data_server_appserver_name`: The name of the directory that will be unzipped by the app server installation bundle, e.g. "`wildfly-8.1.0.Final`".
-* `data_server_appserver_local_dir`: The local directory for the app server installation bundle to be deployed. Must be set if `data_server_artifacts_mode` is "`local`".
-* `data_server_war_name`: The name of the WAR file to be deployed (as saved in either S3 or a local directory).
-* `data_server_war_local_dir`: The local directory of the WAR file to be deployed. Must be set if `data_server_artifacts_mode` is "`local`".
+* `data_server_container`: The path (on the management system) to the app server installation bundle to be deployed.
```

Comment:
I'm assuming we'll need to add this to the jenkins instance(s). This it? https://download.jboss.org/wildfly/8.1.0.Final/wildfly-8.1.0.Final.zip

Reply:
Nope, not quite. Take a look at `build.groovy` and note the `BuildResults` struct it returns. That struct contains the paths to the JARs, WARs, and Wildfly installer. The struct will get passed into the AMI build, so all those artifacts can be used there.

(I just noticed that it _isn't_ being passed in right now, so I'll fix that.)